src/components/network/network-table.js

```diff
@@ -1030,17 +1057,28 @@ const NetworkTable = (props) => {
                                     },
                                     startAdornment: (
                                         <InputAdornment position="start">
-                                            <SearchIcon />
+                                            <SearchIcon
+                                                color={
+                                                    isModifyingRow()
+                                                        ? 'disabled'
+                                                        : ''
+                                                }
+                                            />
                                         </InputAdornment>
                                     ),
                                 }}
                             />
                         </Grid>
                         <Grid item className={classes.selectColumns}>
-                            <span>
+                            <span
+                                className={clsx({
```

Comment:
don't use clsx if you don't compose classes I think

Reply:
I think it's easier to read this way rather than using ternary solution

Reply:
OK